ktlint-cli/build.gradle.kts

```diff
@@ -56,7 +55,9 @@ val ktlintCliFiles by tasks.registering(KtlintCliTask::class) {
     ktlintCliWindowsBatchScriptSource.set(layout.projectDirectory.file("src/main/scripts/ktlint.bat"))
     ktlintCliOutputDirectory.set(ktlintCliOutputRoot)
 
-    finalizedBy("signKtlintCliFiles")
+    if (pluginManager.hasPlugin("signing")) {
```

Comment:
Hmm, maybe instead I should do:
```
val signKtlintCliFiles by tasks.registering(Sign::class) {
   isEnabled = pluginManager.hasPlugin("signing")
```
so the task is being reported as skipped 🤔 I'll need to check how other signing tasks are reported to maintain consistency 👀

Reply:
Ok, I will keep the PR open until you have checked.

Reply:
@mateuszkwiecinski, maybe this escaped your attention. Can you please confirm whether I should wait with merging?

Reply:
My appologies 😬 

I confirmed with the current setup, "regular" signing tasks are not present, so the config I proposed initially has been consistent hence valid 👍 
Link to sample run: https://scans.gradle.com/s/hr4iqnkmfjxmg/timeline